build.gradle

```diff
@@ -20,6 +21,9 @@ dependencies {
     compile 'org.bitbucket.b_c:jose4j:0.4.4'
     compile 'org.bouncycastle:bcprov-jdk15on:1.52'
     compile 'org.bouncycastle:bcpkix-jdk15on:1.52'
+    compile 'com.google.code.gson:gson:1.7.2'
```

Comment:
Can this be replaced by the minimal-json library that the SDK already uses?
We are reluctant to add any new dependencies to the SDK unless absolutely necessary.
Can the pattern Gson().fromJson(response.getJSON(), RetentionPolicyAssignment.Info.class);  by replaced by the existing pattern JsonObject.readFrom(response.getJSON());
I think this would also remove the following two dependencies  org.apache.commons:commons-io and org.apache.commons:commons-lang3